src/components/Navbar/styles.scss

```diff
@@ -15,4 +15,21 @@
   @include respond-above(md) {
     height: rem(72px);
   }
+
+  a {
```

Comment:
Avoid using tags directly if it's not the last option. Would be better to create a class isn't it?